packets.go

```diff
@@ -912,6 +912,12 @@ func (stmt *mysqlStmt) writeExecutePacket(args []driver.Value) error {
 	const minPktLen = 4 + 1 + 4 + 1 + 4
 	mc := stmt.mc
 
+	// Determine threshould dynamically to avoid packet size shortage as possible.
```

Comment:
s/threshould/threshold/

What do you mean by the "as possible"? When possible? As much as possible?
Are there still cases where the driver sends too short packets?

Reply:
> What do you mean by the "as possible"? When possible? As much as possible?

Sorry, I'm not English speaker.  I meant it's OK for most realistic use cases,
although there can be edge cases.

> Are there still cases where the driver sends too short packets?

No, always send more than 64 bytes.
"shortage" meant there is chance which stmt_exec packet is bigger than
max_allowed_packet.
For example, user can set very small max_allowed_packet.  In such case,
packet can be bigger than max_allowed_packet even when all data is sent
with STMT_LONG_DATA.

Reply:
I'm not good English writer.  Could you rewrite this comment?

Reply:
just remove the "as possible"